stream_alert/rule_processor/threat_intel.py

```diff
@@ -61,27 +63,27 @@ def read_compressed_files(self):
                         "0123456789abcdef0123456789abcdef": ["mal_md5", "source_bar"]
                     }
                 }
-            None: if location storing intelligence is not existed
+            None: if the intelligence directory does not exist
         """
         if not os.path.exists(self._intel_dir):
             return
 
-        gz_files = [os.path.join(self._intel_dir, gz_file)
-                    for gz_file in os.listdir(self._intel_dir)
+        gz_files = [os.path.join(self._intel_dir, gz_file) for gz_file
+                    in os.listdir(self._intel_dir)
                     if gz_file.endswith('.gz')]
 
         for gz_file in gz_files:
-            with gzip.open(gz_file, 'rb') as f:
-                csv_reader = csv.reader(f, delimiter=self._delimiter)
+            with gzip.open(gz_file, 'r') as ioc_file:
+                csv_reader = csv.reader(ioc_file, delimiter=self._delimiter)
                 ioc_type = os.path.basename(gz_file).split('.')[0]
-                if ioc_type not in self._intel_dict:
-                    self._intel_dict[ioc_type] = dict()
+                if ioc_type not in self._intel_dictionary:
+                    self._intel_dictionary[ioc_type] = dict()
                 for row in csv_reader:
                     if len(row) < 2:
                         LOGGER.debug('Warning, each row in CSV file should \
```

Comment:
The `\` in the line above should be avoided like so:
```
                    LOGGER.debug('Warning, each row in CSV file should'
                                 'contain at least two fields. Bad row [%s]', row)
```
Also, can the hanging `row)` just be placed on the preceding line, like above ^ ?